.gitignore

```diff
@@ -1,149 +1,274 @@
+
+# Linux libs
+thirdparty/gperftools-2.0/.libs/
+thirdparty/gperftools-2.0/.deps/
+thirdparty/gperftools-2.0/vsprojects/
+thirdparty/gperftools-2.0/**/*.o
+thirdparty/gperftools-2.0/**/*.lo
+thirdparty/gperftools-2.0/**/*.la
+thirdparty/gperftools-2.0/*_test
+thirdparty/gperftools-2.0/*_unittest
+thirdparty/gperftools-2.0/aclocal.m4
+thirdparty/gperftools-2.0/missing
+thirdparty/gperftools-2.0/mkinstalldirs
+thirdparty/gperftools-2.0/compile
+thirdparty/gperftools-2.0/config.guess
+thirdparty/gperftools-2.0/config.sub
+thirdparty/gperftools-2.0/depcomp
+thirdparty/gperftools-2.0/install-sh
+thirdparty/gperftools-2.0/Makefile.in
+thirdparty/gperftools-2.0/test-driver
+thirdparty/gperftools-2.0/Makefile
+thirdparty/gperftools-2.0/config.status
+thirdparty/gperftools-2.0/src/config.h
+thirdparty/gperftools-2.0/libtool
+thirdparty/gperftools-2.0/configure
+thirdparty/gperftools-2.0/*.pc
+
+external/crypto++-*/a.out
+external/crypto++-*/*.o
+
+thirdparty/libedit-3.1/**/Makefile
+thirdparty/libedit-3.1/**/Makefile.in
+thirdparty/libedit-3.1/**/*.o
+thirdparty/libedit-3.1/**/*.lo
+thirdparty/libedit-3.1/**/*.la
+thirdparty/libedit-3.1/Makefile
+thirdparty/libedit-3.1/**/.deps
+thirdparty/libedit-3.1/**/.libs
+thirdparty/libedit-3.1/config.h
+thirdparty/libedit-3.1/config.status
+thirdparty/libedit-3.1/config.guess
+thirdparty/libedit-3.1/config.sub
+thirdparty/libedit-3.1/install-sh
+thirdparty/libedit-3.1/missing
+thirdparty/libedit-3.1/depcomp
+thirdparty/libedit-3.1/compile
+thirdparty/libedit-3.1/*.pc
+thirdparty/libedit-3.1/aclocal.m4
+thirdparty/libedit-3.1/acinclude.m4
+thirdparty/libedit-3.1/autom4te.cache/
+thirdparty/libedit-3.1/configure
+thirdparty/libedit-3.1/libtool
+
+thirdparty/protobuf-*/aclocal.m4
+thirdparty/protobuf-*/gtest/configure
+thirdparty/protobuf-*/configure
+thirdparty/protobuf-*/src/protoc
+thirdparty/protobuf-*/src/unittest_proto_middleman
+thirdparty/protobuf-*/**/Makefile
+thirdparty/protobuf-*/**/Makefile.in
+thirdparty/protobuf-*/**/libtool
+thirdparty/protobuf-*/**/.dirstamp
+thirdparty/protobuf-*/protobuf*.pc
+thirdparty/protobuf-*/gtest/config.status
+thirdparty/protobuf-*/gtest/scripts/gtest-config
+thirdparty/protobuf-*/gtest/build-aux/config.h
+thirdparty/protobuf-*/**/stamp-h1
+thirdparty/protobuf-*/gtest/autom4te.cache/
+thirdparty/protobuf-*/autom4te.cache/
+thirdparty/protobuf-*/config.h
+thirdparty/protobuf-*/config.status
+thirdparty/protobuf-*/test-driver
+thirdparty/protobuf-*/missing
+thirdparty/protobuf-*/**/*.o
+thirdparty/protobuf-*/**/*.la
+thirdparty/protobuf-*/**/*.lo
+thirdparty/protobuf-*/**/.deps
+thirdparty/protobuf-*/**/.libs
+thirdparty/protobuf-*/src/google/protobuf/compiler/cpp/cpp_test_bad_identifiers.pb.*
+thirdparty/protobuf-*/src/google/protobuf/unittest.pb.*
+thirdparty/protobuf-*/src/google/protobuf/.deps/
+thirdparty/protobuf-*/src/google/protobuf/.dirstamp
+thirdparty/protobuf-*/src/google/protobuf/.libs/
+
+# Specific Source build products
+lib/public/linux32/*.so
```

Comment:
Why?

Reply:
...because we don't want to include build products in commits? I don't remember making this change, but it makes sense to me.